openwisp_monitoring/device/tests/test_models.py

```diff
@@ -578,6 +578,90 @@ def _create_env(self):
         )
         return dm, ping, load, process_count
 
+    def test_disabling_critical_check(self):
+
+        Check = load_model('check', 'Check')
+        dm, ping, load, process_count = self._create_env()
+
+        ping_check_instance = Check.objects.create(
+            name='Ping Check',
+            check_type='ping',
+            content_object=dm.device,
+            params={},
+        )
+        dm.update_status('ok')
+
+        self.assertEqual(dm.status, 'ok')
```

Comment:
This line is useless, please remove it.
We already trust `dm.update_status('ok')` to work because there's other tests which ensure that, so there's no reason to check again here. Let's avoid bloating the tests.